# Message for the definition of an Action
std_msgs/Header header
mars_common_msgs/Id robot_id
mars_common_msgs/Id action_id
mars_common_msgs/Id task_id
#definition of the action
mars_agent_physical_robot_msgs/RobotAction action

